src/js/stores/DCOSStore.js

```diff
@@ -34,12 +38,20 @@ class DCOSStore extends EventEmitter {
         serviceTree: new ServiceTree(),
         versions: new Map()
       },
+      chronos: {
+        jobTree: new JobTree(),
```

Comment:
Not quite sure if this is the right way of doing this, as we try to refrain from storing structs in the stores (and potentially the omni store), and the wrap the data in the structs when the are returned from the store.